test/sub-dir.sh

```diff
@@ -1,4 +1,4 @@
-. $(dirname $0)/_functions.sh
+. $(dirname "$0")/_functions.sh
```

Comment:
:+1: could you change `test/config-dir.sh` too?

Reply:
Done.